src/data/helper/dimensionHelper.ts

```diff
@@ -169,6 +173,10 @@ export function summarizeDimensions(
         defaultedLabel = encodeLabel.slice();
     }
 
+    if (!defaultedLabel || !defaultedLabel.length) {
+        defaultedLabel = [lastExtraCoord];
```

Comment:
Should better not give a defaultLabel as `['']`, (the `lastExtraCoord` is initialized with `''`, and there is no non-empty judgement.